components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/edit.jsp

```diff
@@ -391,6 +406,10 @@
                     //on load adjust the form based on the current settings
                     adjustForm();
                     $("form[name='editAppform']").change(adjustForm);
+
+                    // Set selected encryption algorithm and encryption method.
+                    $('select[name=encryptionAlgorithm]').val('<%=Encode.forHtmlAttribute(app.getEncryptionAlgorithm())%>');
+                    $('select[name=encryptionMethod]').val('<%=Encode.forHtmlAttribute(app.getEncryptionMethod())%>');
```

Comment:
Did we check how jquery behaves. Also, this is a JS function. If this is to be encoded it should be for javascript.
Please revalidate

Reply:
Changed for javascript.